test/functionalTest/cases/0981_POST_v2_attributes/append_and_append_only_combinations.test

```diff
@@ -146,7 +146,7 @@ Date: REGEX(.*)
 
 03. POST /v2/entities/E1 attribute 'attr2'
 ==========================================
-HTTP/1.1 201 Created
+HTTP/1.1 204 No Content
 Content-Length: 0
 Location: /v2/entities/E1
```

Comment:
You are right, http://telefonicaid.github.io/fiware-orion/api/v2/ specifies the way you mention. I have double check with @jmcanterafonseca. It will be fixed in a next commit.